Gopkg.toml

```diff
@@ -1,23 +1,23 @@
-[[constraint]]
-  name = "github.com/codahale/hdrhistogram"
-
 [[constraint]]
   name = "github.com/go-kit/kit"
   version = "0.5.0"
 
 [[constraint]]
-  name = "github.com/influxdata/influxdb"
+  name = "github.com/codahale/hdrhistogram"
+  branch = "master"
```

Comment:
Is there some logic behind the reordering?

Reply:
Yes - I reordered it so that the dependencies are in the same order as glide.yaml so that they may be visually compared easier